{"uid":"8a9e6c5f5c77e7af","name":"test_upload_service","fullName":"test_upload_service#test_upload_service","historyId":"325fd6a8ea79205146093fedd5e195bf","time":{"start":1746509151711,"stop":1746509151715,"duration":4},"status":"broken","statusMessage":"FileNotFoundError: [Errno 2] No such file or directory: 'tests/data/sample_file.txt'","statusTrace":"def test_upload_service():\n        client = ApiClient()\n        file_path = os.getenv(\"UPLOAD_FILE_PATH\", \"data/sample_file.txt\")\n>       response = client.upload_file(file_path)\n\ntest_upload_service.py:11: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <tests.api.clients.api_client.ApiClient object at 0x7fd55e59c430>\nfile_path = 'tests/data/sample_file.txt'\n\n    def upload_file(self, file_path):\n>       with open(file_path, \"rb\") as f:\nE       FileNotFoundError: [Errno 2] No such file or directory: 'tests/data/sample_file.txt'\n\n/app/tests/api/clients/api_client.py:8: FileNotFoundError","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[],"afterStages":[],"labels":[{"name":"suite","value":"test_upload_service"},{"name":"host","value":"ca3d992dec03"},{"name":"thread","value":"17-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"test_upload_service"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":true,"retry":true,"extra":{"categories":[],"tags":[]},"source":"8a9e6c5f5c77e7af.json","parameterValues":[]}